enterprise_subsidy/apps/subsidy/migrations/0017_auto_20230905_1511.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.20 on 2023-09-05 15:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('subsidy', '0016_sf_opp_line_item_20230404_1656'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='subsidy',
+            options={'ordering': ['-created']},
+        ),
+        migrations.AddField(
+            model_name='historicalsubsidy',
+            name='is_hidden',
+            field=models.BooleanField(default=False, help_text='If set, this subsidy will be considered as soft-deleted or deactivated.'),
+        ),
+        migrations.AddField(
+            model_name='subsidy',
+            name='is_hidden',
+            field=models.BooleanField(default=False, help_text='If set, this subsidy will be considered as soft-deleted or deactivated.'),
+        ),
+        migrations.AlterField(
+            model_name='historicalsubsidy',
+            name='active_datetime',
+            field=models.DateTimeField(default=None, help_text='The datetime when this Subsidy is considered active.  If null, this Subsidy is considered active.', null=True),
+        ),
+        migrations.AlterField(
+            model_name='historicalsubsidy',
+            name='expiration_datetime',
+            field=models.DateTimeField(default=None, help_text='The datetime when this Subsidy is considered expired.  If null, this Subsidy is considered active.', null=True),
+        ),
+        migrations.AlterField(
+            model_name='subsidy',
+            name='active_datetime',
+            field=models.DateTimeField(default=None, help_text='The datetime when this Subsidy is considered active.  If null, this Subsidy is considered active.', null=True),
```

Comment:
...but also, the existing `help_text` is misleading.  Subsidies with a null `active_datetime` shouldn't be considered active, the code considers them to be _inactive_.  So it'd be good to fix that up when you re-run migrations to add an index on the hidden field.